data/Templates/eCR/Resource/_ObservationEmergencyOutbreakInformation.liquid

```diff
@@ -0,0 +1,32 @@
+{
+    "fullUrl":"urn:uuid:{{ ID }}",
+    "resource":{
+        "resourceType": "Observation",
+        "id":"{{ ID }}",
+        "meta":
+        {
+            "profile":
+            [
+                "http://hl7.org/fhir/us/ecr/StructureDefinition/us-ph-emergency-outbreak-information"
+            ]
+        },
+        "identifier":
+        [
+            {% assign ids = observationEntry.id | to_array -%}
+            {% for i in ids -%}
+                { {% include 'DataType/Identifier' Identifier: i -%} },
```

Comment:
Does our output need to be valid json? I think this for loop will output a trailing comma making for invalid json. Found an example on the shopify liquid [docs](https://shopify.dev/docs/api/liquid/objects/forloop).

```suggestion
                { {% include 'DataType/Identifier' Identifier: i -%} }{% unless forloop.last %},{% endunless %}
```